src/workflows/airqo_etl_utils/datautils.py

```
@@ -1020,7 +1023,7 @@ def extract_data_from_bigquery(
         device_network: Optional[DeviceNetwork] = None,
         dynamic_query: Optional[bool] = False,
         remove_outliers: Optional[bool] = True,
-        data_filter: Optional[Dict[str, Any]] = None,
+        data_filter: Optional[Dict[str, Any]] = {},
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)